repos/fdbt-site/src/pages/ticketRepresentation.tsx

```diff
@@ -90,14 +94,15 @@ export const getServerSideProps = (ctx: NextPageContextWithSession): { props: Ti
     const csrfToken = getCsrfToken(ctx);
     const { fareType } = getSessionAttribute(ctx.req, FARE_TYPE_ATTRIBUTE) as FareType;
     const ticketType = getSessionAttribute(ctx.req, TICKET_REPRESENTATION_ATTRIBUTE);
+    const isCarnet = getSessionAttribute(ctx.req, CARNET_FARE_TYPE_ATTRIBUTE);
 
     return {
         props: {
             fareType,
             errors: ticketType && isTicketRepresentationWithErrors(ticketType) ? ticketType.errors : [],
             csrfToken,
             showHybrid: fareType !== 'multiOperator',
-            showPointToPoint: process.env.STAGE !== 'prod' && fareType !== 'multiOperator',
+            showPointToPoint: process.env.STAGE !== 'prod' && fareType !== 'multiOperator' && !isCarnet,
```

Comment:
Do we want to switch this on for prod at this point?
```suggestion
            showPointToPoint: fareType !== 'multiOperator' && !isCarnet,
```

Reply:
Yeah was going to let the UI tests run on test first but will follow up with this straight after 🎉